packages/phpstan-rules/tests/Rules/DeadCode/UnusedPublicClassConstRule/Source/ConstantUser.php

```diff
@@ -4,11 +4,16 @@
 namespace Symplify\PHPStanRules\Tests\Rules\DeadCode\UnusedPublicClassConstRule\Source;
 
 use Symplify\PHPStanRules\Tests\Rules\DeadCode\UnusedPublicClassConstRule\Fixture\SkipUsedPublicConstant;
+use Symplify\PHPStanRules\Tests\Rules\DeadCode\UnusedPublicClassConstRule\Fixture\SkipUsedPublicConstantInSubclass;
 
 final class ConstantUser
 {
     public function run()
     {
         return SkipUsedPublicConstant::USED;
     }
+    public function run2()
+    {
+        return SkipUsedPublicConstantInSubclass::USED_FROM_SUBCLASS;
```

Comment:
this one is the actualy case, I wanted to add

Reply:
This should be moved to new fixture file, to isolate normal + nested constant tests.

Reply:
done